src/main/java/com/example/flipcardsapp/flipCard/dao/model/FlipCard.java

```diff
@@ -1,5 +1,77 @@
 package com.example.flipcardsapp.flipCard.dao.model;
 
-public interface FlipCard {
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
 
+@Entity
+public class FlipCard {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private float studyPercent;
+    private String uuid;
+
+    @OneToOne(cascade = CascadeType.PERSIST)
+
+    private FlipCardFront flipCardFront;
+    @OneToOne(cascade = CascadeType.PERSIST)
+    private FlipCardBack flipCardBack;
+
+    public FlipCard() {
+    }
+
+    public FlipCard(FlipCardFront flipCardFront, FlipCardBack flipCardBack) {
+        this.flipCardFront = flipCardFront;
+        this.flipCardBack = flipCardBack;
+        this.studyPercent = 0.01f;
+        this.uuid = UUID.randomUUID().toString();
+    }
+
+    public FlipCard(float studyPercent, UUID uuid, FlipCardFront flipCardFront, FlipCardBack flipCardBack) {
+        this.studyPercent = studyPercent;
+        this.uuid = uuid.toString();
+        this.flipCardFront = flipCardFront;
+        this.flipCardBack = flipCardBack;
+    }
```

Comment:
you are using two different constructors in project? For what? 
only for my information

Reply:
the first one is for ProdDB class to mockup some data, the second one is for flip cards from front application where study percent and uuid will be assigned before it will be saved in db